### PR TITLE
Docs: Checked wiki for spelling errors

### PR DESCRIPTION
**Pull request by NootSoGreen**
Cursory check of wiki (-Boards section) for spelling errors.

### CODE REVIEW

File: docs/wiki/guides/archive/Single-Wire-Software-Serial.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ OMNIBUS(F3) (by @jflyper)
 | --- | --------- | ------- | ----- | ------ | ----------------------------------------------- |
 | A8  | LED strip | NG      | NG    | NG     |                                                 |
 | B4  | PPM (\*1) | OK      | ?     | OK     | When PPM not in use                             |
-| B6  | PWM8/SCL  | OK      | OK    | OK     | I2C must be de-configured? Need furthe testing. |
+| B6  | PWM8/SCL  | OK      | OK    | OK     | I2C must be de-configured? Need further testing. |
```
Comment (haslinghuis) [inline docs/wiki/guides/archive/Single-Wire-Software-Serial.md:37]:
```suggestion
| B6  | PWM8/SCL  | OK      | OK    | OK     | I2C must be de-configured? Need further testing.|
```

File: docs/wiki/guides/current/Magnetometer.md [hunk 2]
```diff
@@ -168,7 +168,7 @@ No matter how the magnetometer is mounted, is absolutely essential to confirm pr
...
+When the Mag orientation (alignment) is correct, the 'quad icon' in the Home screen of Configurator should move smoothly and appropriately as the quad is rotated, pitched and rolled. Note that when the quad's attitude is changed very quickly, the heading will initially react quickly, using Gyro and Acc data, and then over the next half second will be adjusted according to the Mag data. If the orientation of the Mag data axes is wrong, or the calibration is off, then the 'quad icon' will turn correctly at first, but then quickly shift to an incorrect heading. That's a red flag; you'll need to fix the Orientation, and re-calibrate.
```
Comment (haslinghuis) [inline docs/wiki/guides/current/Magnetometer.md:171]:
```suggestion
When the Mag orientation (alignment) is correct, the 'quad icon' in the Home screen of Configurator should move smoothly and appropriately as the quad is rotated, pitched and rolled. Note that when the quad's attitude is changed very quickly, the heading will initially react quickly, using Gyro and Acc data, and then over the next half second will be adjusted according to the Mag data. If the orientation of the Mag data axes is wrong, or the calibration is off, then the 'quad icon' will turn correctly at first, but then quickly shift to an incorrect heading. That's a red flag; you'll need to fix the orientation, and re-calibrate.
```

File: docs/wiki/guides/current/VTX-CLI-Settings.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ As of Betaflight version 3.3.0, the CLI settings below can be used to
 configure addressable video transmitters (such as
 TBS-[SmartAudio](/docs/wiki/guides/current/SmartAudio)
 and
-IRC-[Tramp](IRC-Tramp)[)](IRC-Tramp)
+IRC-[Tramp](IRC-Tramp))
```
Comment (haslinghuis) [inline docs/wiki/guides/current/VTX-CLI-Settings.md:7]:
```suggestion
IRC-[Tramp](IRC-Tramp)
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
Thank you !

File: docs/wiki/guides/current/VTX-CLI-Settings.md (hunk 3 above)
Comment (NootSoGreen) [reply docs/wiki/guides/current/VTX-CLI-Settings.md:7]: The bracket you're sugesting to remove is paired with the one on line 4

> configure addressable video transmitters **(**such as

**Review by haslinghuis — APPROVED** [review]

**Review by VitroidFPV — APPROVED** [review]

**Pull request merged**